build/charts/antrea/templates/agent/clusterrole.yaml

```diff
@@ -180,3 +180,17 @@ rules:
       - get
       - list
       - watch
+  {{- if eq .Values.trafficEncryptionMode "ipsec" }}
```

Comment:
Currently all permissions are added by default regardless of whether they are used. It may be not the best but I think better to keep consistency for now. One of the benefits is that users don't have to update clusterrole when enabling ipsec.

Reply:
ok. updated.